tests/test_frictionless_to_ckan.py

```diff
@@ -123,24 +129,27 @@ def test_dataset_license(self):
             'licenses': [
                 {
                     'title': 'Open Data Commons Open Database License',
-                    'type': 'odc-pddl'
+                    'name': 'odc-pddl'
```

Comment:
Available properties for a given license according to the specs are `name`, `title` and `path`.
* https://specs.frictionlessdata.io/schemas/data-package.json
* https://specs.frictionlessdata.io/data-package/#licenses